Dockerfile

```diff
@@ -23,11 +23,18 @@ RUN apk add --no-cache ca-certificates
 RUN addgroup -g 65532 -S nonroot && adduser -u 65532 -S nonroot -G nonroot
 USER 65532
 
+FROM alpine:3.15.4 AS ssl_git_runner
```

Comment:
Why don't you inherit from `ssl_runner` and only add git?

Reply:
`apk add` does not work anymore after `USER 65532` is set and there is no `sudo` in alpine installed by default.
Having a new image looked cleaner for me comparing to switching `USER` back and forth.